code/datums/status_effects/buffs.dm

```diff
@@ -40,6 +40,132 @@
 	playsound(owner, 'sound/weapons/bite.ogg', 50, 1)
 	owner.adjustBruteLoss(3)
 
+/datum/status_effect/blooddrunk
```

Comment:
I may need to tweak this; I really don't like TG's implementation of it, and it's going to work strangely, in some edge cases, with broken bones and such.